styles/theme.css

```diff
@@ -3,49 +3,99 @@
 @tailwind utilities;
 
 @layer utilities {
-  display-1-bold {
-    @apply font-size: 32px line-height:40px font-weight: 700;
+  .auth-background {
+    @apply min-w-screen min-h-screen bg-black-900;
```

Comment:
```suggestion
  .auth-background {
    @apply h-screen bg-black-900  flex flex-col items-center justify-start gap-y-40 pt-[72px];
```
`min-w-screen` is not a class. Let's use a full screen height for this, and let's apply all the padding for the page and flex here on the layout's class instead of elsewhere and inside of child divs.

![image](https://github.com/agordon123/git_note/assets/4108484/ab1edbb4-ee5a-48e4-be02-b79796a3b8bd)


If we look at the figma, the layout of the onboarding and auth pages are roughly the same except for the gap-y, which varies slightly. Let's find a good medium distance for the gap instead of making 4 different gap values.

In this case, I chose gap-y-40, you might find a little less or more is nicer.

Reply:
noted and thank you